rero_ils/modules/acq_accounts/jsonschemas/acq_accounts/acq_account-v0.0.1.json

```diff
@@ -0,0 +1,89 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "Acquisition account",
+  "description": "JSON schema for an acquisition account",
+  "additionalProperties": false,
+  "required": [
+    "$schema",
+    "pid",
+    "name",
+    "currency",
+    "fiscal_year",
+    "amount_allocated",
+    "library"
+  ],
+  "properties": {
+    "$schema": {
+      "title": "Schema",
+      "description": "Schema to validate account records against.",
+      "type": "string",
+      "minLength": 9,
+      "default": "https://ils.rero.ch/schema/acq_accounts/acq_account-v0.0.1.json"
+    },
+    "pid": {
+      "title": "Account ID",
+      "type": "string"
+    },
+    "name": {
+      "title": "Name",
+      "description": "Name of the Acquisition account.",
+      "type": "string",
+      "minLength": 1
+    },
+    "description": {
+      "title": "Description",
+      "description": "Description of the Acquisition account.",
+      "type": "string",
+      "minLength": 1
+    },
+    "currency": {
+      "title": "Currency",
+      "description": "Code of the currency.",
+      "type": "string",
+      "enum": [
+        "CHF",
+        "EUR",
+        "USD"
+      ]
+    },
+    "fiscal_year": {
+      "title": "Fiscal year",
+      "description": "The Acquisition account belongs to which fiscal year.",
+      "type": "string",
```

Comment:
Why is not a number ?

Reply:
Because the fiscal year is not necessarily `Jan` -> `Dec`. some libraries they starts in July and ends in June of next year.